src/webui/src/components/TrialsDetail.tsx

```diff
@@ -241,13 +255,19 @@ class TrialsDetail extends React.Component<{}, TrialDetailState> {
     handleEntriesSelect = (value: string) => {
         switch (value) {
             case '20':
-                this.setState(() => ({ entriesTable: 20 }));
+                if (this._isMounted) {
+                    this.setState(() => ({ entriesTable: 20 }));
+                }
                 break;
             case '50':
-                this.setState(() => ({ entriesTable: 50 }));
+                if (this._isMounted) {
+                    this.setState(() => ({ entriesTable: 50 }));
+                }
                 break;
             case '100':
-                this.setState(() => ({ entriesTable: 100 }));
+                if (this._isMounted) {
```

Comment:
It seems unnecessary to use 3 cases for 20, 50 and 100, these 3 cases can be combined into this.setState(() => ({ entriesTable: value }));

Reply:
Ok, I'll change. Thanks!